storage/file/file.go

```diff
@@ -72,3 +73,14 @@ func (p *Provider) Store(url, alias string) error {
 
 	return nil
 }
+
+func (p *Provider) DeleteURL(alias string) error {
+	exists, _ := p.Exists(alias)
```

Comment:
```suggestion
	exists, err := p.Exists(alias)
	if err != nil {
		return fmt.Errorf("checking if alias exists: %v", err)
	}
```

Let's bubble up errors from `Exists()` similar to the other providers.

As a note, you would usually [wrap an error using `%w`](https://www.digitalocean.com/community/tutorials/how-to-add-extra-information-to-errors-in-go#wrapping-and-unwrapping-errors) instead of `%v`, but that was added in Go 1.13 and the klein repo is still on 1.12 as it hasn't really been given any attention in a long time😅 So we'll stick with `%v` for now.